functions/Copy-DbaDbMail.ps1

```diff
@@ -264,6 +264,13 @@ function Copy-DbaDbMail {
             $sourceMailServers = $sourceServer.Mail.Accounts.MailServers
             $destMailServers = $destServer.Mail.Accounts.MailServers
 
+            Write-Message -Message "Getting mail server credentials." -Level Verbose
+            $credentialAccounts = @($sourceServer.Query("SELECT credentials.name AS credential_name, sysmail_server.account_id FROM sys.credentials
```

Comment:
Maybe first assign $sql in one line and then use it in the second?

Reply:
I have pushed a new commit to resolve the formatting issues. Thanks, I learned something new today about running Invoke-DbatoolsFormatter and dbatools.Tests.ps1 :)